### ISSUE TITLE
missing ANXS role?

### ISSUE THREAD

**Issue by boblannon** [sunlightlabs/kafka-examples#1]
got an error running vagrant up:

```
ERROR: cannot find role in /home/blannon/dev/kafka-examples/ansible/roles/ANXS.postgresql or /home/blannon/dev/kafka-examples/ansible/ANXS.postgresql or /etc/ansible/roles/ANXS.postgresql
Ansible failed to complete successfully. Any error output should be
visible above. Please fix these errors and try again.
```

**Comment by zmaril** [comment]
I used [ansible galaxy](https://galaxy.ansible.com/) for this, though it was a while ago and forgot to include it in readme. I'll add it in and remove the submodule.

**Comment by zmaril** [comment]
Closed by 0012141e4c2f1461bd508507b77f37899246b69b
